salt/modules/zk_concurrency.py

```diff
@@ -276,6 +277,8 @@ def unlock(path,
 
 def party_members(path,
                   zk_hosts,
+                  min_nodes,
```

Comment:
Good catch, I was very careful about the states/zk_concurrency.py backward compatibility but not so much about the modules/zk_concurrency.py compatibility since I did not see many functions calling this function. Will make these changes along with doc and update the PR.